src/Sarif.Viewer.VisualStudio.Core/Options/SarifViewerOptionsControlResources.xaml

```diff
@@ -2,7 +2,8 @@
                     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
                     xmlns:system="clr-namespace:System;assembly=mscorlib">
 
-    <system:String x:Key="SarifViewerOptionsControl_GroupBoxHeader">General</system:String>
+    <system:String x:Key="SarifViewerOptionsControl_GeneralGroupBoxHeader">General</system:String>
+    <system:String x:Key="SarifViewerOptionsControl_DevCanvasGroupBoxHeader">DevCanvas</system:String>
```

Comment:
Why couldn't this be done in the extension's reserved space? I wanted to avoid this kind of leakage.

Reply:
What do you mean reserved space, should I setup a separate .xaml resources for the strings that we use for the devcanvas settings?

Reply:
The result source platform supports this, see GitHubSourceService.InitializeAsync as an example.

Reply:
I thought that GitHubSourceService's InitializeAsync method and the corresponding processing in ResultSourceHost_ServiceEvent creates new options for when you right click on error list items, not the options menu which this affects.
I also see that [SarifViewerOptionsControlResources.xaml](https://github.com/microsoft/sarif-visualstudio-extension/pull/602/files/9e1f67abf6a62bfc1c95d2113be5b8d4ab2a49d6#diff-d2cfa9f8d8b3ad366f111020ae0fc4098f35c7c97dd579c37f930d2b61934e6b) includes lines for the GhAS result source service on line 12 with "<system:String x:Key="SarifViewerOptionsControl_EnableGitHubAdvancedSecurity">Enable GitHub Advanced Security scan results</system:String>"

Reply:
Ahh sorry, I misunderstood where the change was. This is fine; no reasonable way to do it cleaner. The only leakage I couldn't avoid!